src/node_errors.cc

```diff
@@ -140,8 +140,43 @@ static std::string GetErrorSource(Isolate* isolate,
   return std::string(buf);
 }
 
+void PrintStackTrace(Isolate* isolate) {
+  Local<Context> context = isolate->GetCurrentContext();
+  Local<String> error_string =
+      String::NewFromOneByte(isolate,
+                             reinterpret_cast<const uint8_t*>("Error"),
+                             v8::NewStringType::kInternalized,
+                             5)
+          .ToLocalChecked();
+  Local<String> stack_trace_limit_string =
+      String::NewFromOneByte(
+          isolate,
+          reinterpret_cast<const uint8_t*>("stackTraceLimit"),
+          v8::NewStringType::kInternalized,
+          15)
+          .ToLocalChecked();
+  MaybeLocal<Value> stack_trace_limit_val =
+      context->Global()
+          ->Get(context, error_string)
+          .ToLocalChecked()
+          .As<Object>()
+          ->Get(context, stack_trace_limit_string);
```

Comment:
Perhaps we can use `env->primordials()` to get `ErrorStackTraceLimit`, though I'm not sure if it's appropriate here?

Reply:
That would only give the stack trace limit at the time of process startup. I think this is the correct approach in general